pkg/minikube/out/out_reason.go

```diff
@@ -71,33 +70,42 @@ func displayText(k reason.Kind, format string, a ...V) {
 		st = style.KnownIssue
 	}
 
-	ErrT(st, format, a...)
+	determineOutput(st, format, a...)
 
 	if k.Advice != "" {
 
 		advice := indentMultiLine(Fmt(k.Advice, a...))
-		ErrT(style.Tip, Fmt("Suggestion: {{.advice}}", V{"advice": advice}))
+		determineOutput(style.Tip, Fmt("Suggestion: {{.advice}}", V{"advice": advice}))
 	}
 
 	if k.URL != "" {
-		ErrT(style.Documentation, "Documentation: {{.url}}", V{"url": k.URL})
+		determineOutput(style.Documentation, "Documentation: {{.url}}", V{"url": k.URL})
 	}
 
 	issueURLs := k.IssueURLs()
 	if len(issueURLs) == 1 {
-		ErrT(style.Issues, "Related issue: {{.url}}", V{"url": issueURLs[0]})
+		determineOutput(style.Issues, "Related issue: {{.url}}", V{"url": issueURLs[0]})
 	}
 
 	if len(issueURLs) > 1 {
-		ErrT(style.Issues, "Related issues:")
+		determineOutput(style.Issues, "Related issues:")
 		for _, i := range issueURLs {
-			ErrT(style.Issue, "{{.url}}", V{"url": i})
+			determineOutput(style.Issue, "{{.url}}", V{"url": i})
 		}
 	}
 
 	if k.NewIssueLink {
-		ErrT(style.Empty, "")
+		determineOutput(style.Empty, "")
 		displayGitHubIssueMessage()
 	}
 	Ln("")
 }
+
+func determineOutput(st style.Enum, format string, a ...V) {
+	if !JSON {
+		ErrT(st, format, a...)
+		return
+	}
+	errStyled, _ := stylized(st, useColor, format, a...)
+	klog.Warning(errStyled)
```

Comment:
Although we are handling an error, are we ok to log as warning in this case?

Reply:
`klog.Error` will output the string to stdout in raw text form, which we want to avoid when specifying JSON output, hence the `klo.gWarning` which doesn't

Reply:
Followed up with @spowelljr via chat: he confirmed that using an error in this case would return raw text, which is what we are trying to avoid when specifying JSON, therefore we are falling back to using `klog.Warning`.